diff -Nur b.awstats-6.6/tools/awstats_buildstaticpages.pl n.awstats-6.6/tools/awstats_buildstaticpages.pl
--- b.awstats-6.6/tools/awstats_buildstaticpages.pl	2006-02-08 23:26:14.000000000 +0100
+++ n.awstats-6.6/tools/awstats_buildstaticpages.pl	2007-01-16 13:15:15.000000000 +0100
@@ -133,7 +133,7 @@
 	my @PossibleConfigDir=();
 
 	if ($configdir) { @PossibleConfigDir=("$configdir"); }
-	else { @PossibleConfigDir=("$AwstatsDir","$DIR","/etc/awstats","/usr/local/etc/awstats","/etc","/etc/opt/awstats"); }
+	else { @PossibleConfigDir=("$AwstatsDir","$DIR","/etc/webapps/awstats","/etc/awstats","/usr/local/etc/awstats","/etc","/etc/opt/awstats"); }
 
 	# Open config file
 	$FileConfig=$FileSuffix='';
@@ -293,8 +293,8 @@
 # Check if AWSTATS prog is found
 my $AwstatsFound=0;
 if (-s "$Awstats") { $AwstatsFound=1; }
-elsif (-s "/usr/local/awstats/wwwroot/cgi-bin/awstats.pl") {
-	$Awstats="/usr/local/awstats/wwwroot/cgi-bin/awstats.pl";
+elsif (-s "/usr/share/awstats/wwwroot/cgi-bin/awstats.pl") {
+	$Awstats="/usr/share/awstats/wwwroot/cgi-bin/awstats.pl";
 	$AwstatsFound=1;
 }
 if (! $AwstatsFound) {
diff -Nur b.awstats-6.6/tools/awstats_configure.pl n.awstats-6.6/tools/awstats_configure.pl
--- b.awstats-6.6/tools/awstats_configure.pl	2006-02-08 23:26:14.000000000 +0100
+++ n.awstats-6.6/tools/awstats_configure.pl	2007-01-16 13:15:15.000000000 +0100
@@ -29,11 +29,11 @@
 $AWSTATS_DIRDATA_PATH
 /;
 $AWSTATS_PATH='';
-$AWSTATS_ICON_PATH='/usr/local/awstats/wwwroot/icon';
-$AWSTATS_CSS_PATH='/usr/local/awstats/wwwroot/css';
-$AWSTATS_CLASSES_PATH='/usr/local/awstats/wwwroot/classes';
-$AWSTATS_CGI_PATH='/usr/local/awstats/wwwroot/cgi-bin';
-$AWSTATS_MODEL_CONFIG='/etc/awstats/awstats.model.conf';		# Used only when configure ran on linux
+$AWSTATS_ICON_PATH='/usr/share/awstats/wwwroot/icon';
+$AWSTATS_CSS_PATH='/usr/share/awstats/wwwroot/css';
+$AWSTATS_CLASSES_PATH='/usr/share/awstats/wwwroot/classes';
+$AWSTATS_CGI_PATH='/usr/share/awstats/wwwroot/cgi-bin';
+$AWSTATS_MODEL_CONFIG='/etc/webapps/awstats/awstats.model.conf';		# Used only when configure ran on linux
 $AWSTATS_DIRDATA_PATH='/var/lib/awstats';						# Used only when configure ran on linux
 
 
@@ -608,7 +608,7 @@
 		my $bidon='';
 		print "Directory path to store config file(s) (Enter for default):\n> ";
 		$bidon=<STDIN>; chomp $bidon;
-		if (! $bidon) { $bidon = "/etc/awstats"; }
+		if (! $bidon) { $bidon = "/etc/webapps/awstats"; }
 		my $configdir=$bidon;
 		if (! -d $configdir) {
 			# Create the directory for config files
diff -Nur b.awstats-6.6/tools/awstats_updateall.pl n.awstats-6.6/tools/awstats_updateall.pl
--- b.awstats-6.6/tools/awstats_updateall.pl	2006-07-24 01:57:48.000000000 +0200
+++ n.awstats-6.6/tools/awstats_updateall.pl	2007-01-16 13:15:15.000000000 +0100
@@ -13,7 +13,7 @@
 my $VERSION="1.0 (build $REVISION)";
 
 # Default value of DIRCONFIG
-my $DIRCONFIG = "/etc/awstats";
+my $DIRCONFIG = "/etc/webapps/awstats";
 
 my $Debug=0;
 
@@ -130,8 +130,8 @@
 	# Check if AWSTATS prog is found
 	my $AwstatsFound=0;
 	if (-s "$Awstats") { $AwstatsFound=1; }
-	elsif (-s "/usr/local/awstats/wwwroot/cgi-bin/awstats.pl") {
-		$Awstats="/usr/local/awstats/wwwroot/cgi-bin/awstats.pl";
+	elsif (-s "/usr/share/awstats/wwwroot/cgi-bin/awstats.pl") {
+		$Awstats="/usr/share/awstats/wwwroot/cgi-bin/awstats.pl";
 		$AwstatsFound=1;
 	}
 	if (! $AwstatsFound) {
--- awstats-6.9/wwwroot/cgi-bin/awstats.pl.fix	2009-03-20 13:48:27.572771826 +0100
+++ awstats-6.9/wwwroot/cgi-bin/awstats.pl	2009-03-20 13:49:16.911789620 +0100
@@ -1642,7 +1642,7 @@ sub Read_Config {
 	}
 	else {
 		@PossibleConfigDir = (
-							   "$DIR",                   "/etc/awstats",
+							   "$DIR", "/etc/webapps/awstats", "/etc/awstats",
 							   "/usr/local/etc/awstats", "/etc",
 							   "/etc/opt/awstats"
 		);
