--- wwwroot/cgi-bin/awstats.model.conf~	Fri May 30 13:55:05 2003
+++ wwwroot/cgi-bin/awstats.model.conf	Fri May 30 13:58:36 2003
@@ -748,7 +748,7 @@
 # Example: "/usr/share/awstats/lang"
 # Default: "./lang" (means lang directory is in same location than awstats.pl)
 #
-DirLang="./lang"
+DirLang="/usr/share/awstats/lang"
 
 
 # You choose here which reports you want to see in the main page and what you
